src/pymovements/gaze/_utils/parsing.py

```diff
@@ -192,26 +218,52 @@ def get_pattern_keys(compiled_patterns: list[dict[str, Any]], pattern_key: str)
 
 
 def parse_eyelink_event_start(line: str) -> str | None:
-    """Check if the line contains the start of an event and return the event name."""
-    if FIXATION_START_REGEX.match(line):
-        return 'fixation'
-    if SACCADE_START_REGEX.match(line):
-        return 'saccade'
-    if BLINK_START_REGEX.match(line):
-        return 'blink'
+    """Check if the line contains the start of an event and return the event name.
+
+    Returns event names with eye suffixes: e.g. 'fixation_left' or 'saccade_right'.
+    """
+    if match := FIXATION_START_REGEX.match(line):
+        eye = match.group('eye').upper()
+        suffix = '_left' if eye == 'L' else '_right'
+        return f'fixation{suffix}'
+    if match := SACCADE_START_REGEX.match(line):
+        eye = match.group('eye').upper()
+        suffix = '_left' if eye == 'L' else '_right'
+        return f'saccade{suffix}'
+    if match := BLINK_START_REGEX.match(line):
+        eye = match.group('eye').upper()
+        suffix = '_left' if eye == 'L' else '_right'
+        return f'blink{suffix}'
     return None
 
 
 def parse_eyelink_event_end(line: str) -> tuple[str, float, float] | None:
```

Comment:
This would then be changed to

```python
def parse_eyelink_event_end(line: str) -> tuple[str, str, float, float] | None:
```

Alternatively you could also return a dictionary, but as this is an internal function I think a tuple is perfectly fine.